great_expectations/rule_based_profiler/config/base.py

```diff
@@ -220,14 +220,18 @@ class RuleBasedProfilerConfig(BaseYamlConfig):
     def __init__(
         self,
         name: str,
+        class_name: str,
         config_version: float,
         rules: Dict[str, RuleConfig],
+        module_name: Optional[str] = None,
```

Comment:
@cdkini very minor: could we please keep `class_name` and `module_name` adjacent (in the same order as we have done for Checkpoint config)?  Thanks!

Reply:
class_name is mandatory in this schema so I don't think it'll line up perfectly. I've moved class_name to the bottom of the required params and module_name to the top of the optional params to make them aligned.

Does that suffice or would you like an alternative setup?